### ISSUE TITLE
Sum-of-Squares Types in *Lm functions

### ISSUE THREAD

**Issue by gdevenyi** [Mouse-Imaging-Centre/RMINC#256]
We were recently discussing some more intricacies of linear models and came across the discussions of the lm  sum of squares default.

So, the question arises, what are the sum-of-squares choices for the *lm functions in RMINC, since some of the code underneath the models is custom.

Thanks for the clarifications :)

**Comment by cfhammill** [comment]
I assume you mean for ANOVA type analyses?

**Comment by gdevenyi** [comment]
Right, so after more reading. I see what I missed, this type of question applies to the anova run *on* an lm, which isn't the kind of thing we do here.

But yes, it would be helpful to know which sum of squares type is used for the anova functions.
